### DISCUSSION TITLE
[Feature]邮箱支持

### DISCUSSION THREAD

**Discussion by Fgaoxing** [walinejs/waline#726] [Q&A] [answered]
**功能概述 | Describe the feature**

应该加个判断邮箱商自动改为小写，还有可以在登陆后更改邮箱，加入域名邮箱支持

**Comment by lizheming** [comment]
可以自定义配置 GRAVATAR_STR 为 `https://seccdn.libravatar.org/avatar/{{mail|lower|md5}}` 配置见文档 https://waline.js.org/reference/server.html#%E7%8E%AF%E5%A2%83%E5%8F%98%E9%87%8F

**Comment by Fgaoxing** [comment]
@lizheming 应该加个判断邮箱商自动改为小写，还有可以在登陆后更改邮箱，加入域名邮箱支持

**Comment by Fgaoxing** [comment]
@lizheming 你没明白，不是头像

**Comment by Fgaoxing** [comment]
是邮箱问题！

**Comment by Mister-Hope** [comment]
邮箱什么问题？自己去查w3c文档规定，邮箱可以区分大小写。

**Comment by lizheming** [comment, marked as answer]
邮箱是否小写这个系统不会更改，建议用户自己把控。如果需要修改邮箱目前可以自己去数据库修改。我们没有对域名邮箱做限制，不存在对域名邮箱的支持一说。发消息建议合并一下讨论，保证一下消息质量，谢谢。

**Reply by Fgaoxing** [reply]
主要我想使用admin@yt-blog.top

**Reply by Fgaoxing** [reply]
我要是用admin@yt-blog.top，会说邮箱无效

**Reply by Mister-Hope** [reply]
@lizheming  CC 服务端这里注册有对邮箱做额外判断么？我没找到

**Reply by lizheming** [reply]
> @lizheming CC 服务端这里注册有对邮箱做额外判断么？我没找到

该问题已经处理，可以忽略
